builds/flu/flu_titers.py

```diff
@@ -189,7 +206,7 @@ def glycosylation_count(aa):
     # }
 
 
-def H3N2_scores(runner, tree, epitope_mask_file, epitope_mask_version='wolf'):
+def IAV_scores(runner, tree, mask_file, epitope_mask_version='wolf'):
```

Comment:
I agree with @huddlej that it would be a big improvement to move "scores" outside of `flu_titers` into its own class. Its current placement has always been confusing to me.

Reply:
yes, current placement is pretty much an accident. happy if redesigned.